File: a_system/src/main/java/com/example/a_system/vo/StatisticCourse.java
package com.example.a_system.vo;

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;

public class StatisticCourse {
    private String cno;
    private String cname;
    private String teacher;
    private String credit;
    private int count;  //选该门课的人数

    public StatisticCourse(String cno, String cname, String teacher, String credit, int count) {
        this.cno = cno;
        this.cname = cname;
        this.teacher = teacher;
        this.credit = credit;
        this.count = count;
    }

    @JacksonXmlProperty(localName = "课程编号A")
    public String getCno() {
        return cno;
    }

    public void setCno(String cno) {
        this.cno = cno;
    }

    @JacksonXmlProperty(localName = "课程名A")
    public String getCname() {
        return cname;
    }

    public void setCname(String cname) {
        this.cname = cname;
    }

    @JacksonXmlProperty(localName = "授课教师A")
    public String getTeacher() {
        return teacher;
    }

    public void setTeacher(String teacher) {
        this.teacher = teacher;
    }

    @JacksonXmlProperty(localName = "课程学分A")
    public String getCredit() {
        return credit;
    }

    public void setCredit(String credit) {
        this.credit = credit;
    }

    @JacksonXmlProperty(localName = "选课人数A")
    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }
}

File: system_b/educationalSystem_B/vo/statisticCourse.py

class StatisticCourse:
    def __init__(self, id, name,credit,teacher,count):
        self.id = id
        self.name = name
        self.teacher = teacher
        self.credit = credit
        self.count = count

    def getId(self):
        return self.id

    def setId(self, id):
        self.id = id

    def getName(self):
        return self.name

    def setName(self, name):
        self.name = name

    def getCredit(self):
        return self.credit

    def setCredit(self, credit):
        self.credit = credit

    def getTeacher(self):
        return self.teacher

    def setTeacher(self, teacher):
        self.teacher = teacher

    def getCount(self):
        return self.count

    def setCount(self, count):
        self.count = count